source/coffee/image-picker.coffee

```diff
@@ -26,14 +26,11 @@ sanitized_options = (opts) ->
   jQuery.extend(default_options, opts)
 
 both_array_are_equal = (a,b) ->
-  if (!a || !b) || (a.length != b.length)
-    return false
-  b = b[..]
-  for x in a
-    i = b.indexOf x
-    if i < 0
-      return false
-    b.splice(i, 1)
+  return false if (!a || !b) || (a.length != b.length)
```

Comment:
Dunno why but just got an email regarding this!

I think one downside of this is that `.sort` is an inplace operation. ie, this modifies the original arrays. Need to check the code to see if this has any impact on the rest of the code.

```
var a = [3,2,1]
var b = [3,2,4]
function sortArrays(a, b) {
    a.sort()
    b.sort()
}

console.log(a,b)
>>> [3, 2, 1] [3, 2, 4]
sortArrays(a,b)
console.log(a,b)
>>> [1, 2, 3] [2, 3, 4]
```